erizo/src/erizo/rtp/QualityManager.cpp

```diff
@@ -166,24 +167,27 @@ void QualityManager::selectLayer(bool try_higher_layers) {
     aux_spatial_layer++;
   }
 
-  if (below_min_layer != slideshow_fallback_active_) {
+  ELOG_DEBUG("below_min_layer %u, freeze_fallback_active_: %u", below_min_layer, freeze_fallback_active_);
+  if (below_min_layer != freeze_fallback_active_) {
     if (below_min_layer || try_higher_layers) {
-      slideshow_fallback_active_ = below_min_layer;
+      freeze_fallback_active_ = below_min_layer;
       ELOG_DEBUG("message: Setting slideshow fallback, below_min_layer %u, spatial_layer %d,"
-          "next_spatial_layer %d slidehow_fallback_active_: %d",
-          below_min_layer, spatial_layer_, next_spatial_layer, slideshow_fallback_active_);
+          "next_spatial_layer %d slidehow_fallback_active_: %d, min_desired_spatial_layer: %d",
```

Comment:
shouldn't this be called  `min_requested_spatial` like you do above?

Reply:
Right, actually I think I want both `min_requested_spatial_layer` and `slideshow_below_spatial_layer_` to debug possible problems